Source/WebCore/animation/ViewTimeline.h

```diff
@@ -59,7 +59,18 @@ class ViewTimeline final : public ScrollTimeline {
     AnimationTimeline::ShouldUpdateAnimationsAndSendEvents documentWillUpdateAnimationsAndSendEvents() override;
     AnimationTimelinesController* controller() const override;
 
+    RenderBox* sourceRenderer() const;
+    Element* source() const override;
+
 private:
+    struct Data {
+        float scrollContainerSize = 0;
+        float subjectOffset = 0;
+        float currentScrollOffset = 0;
+        float coverRangeEnd = 0;
```

Comment:
I think that the `{ 0 }` style is preferred, at least that's where the recent webkit-dev conversation is pointing to.

Reply:
Fixed.